src/main/java/eggventory/logic/parsers/ParseEdit.java

```diff
@@ -35,10 +37,15 @@ private Command processEditStock(String input) throws BadInputException {
     private Command processEditStockType(String input) throws BadInputException {
         String[] addInput = input.split(" +", 2);
 
-        if (Parser.isReserved(addInput[1])) {
+
+        if (Parser.isReserved(addInput[0])) {
```

Comment:
isn't the first word like... what kind thing you want to edit? 
not sure if checking isReserved is correct here

Reply:
When I was writing tests, I was getting Exception when I tried the border case of putting a reserved keyword example "edit stock all arg2". Thats why I added this check

Reply:
oh i misread the code. yes you're correct, my bad